Streaming.html

```diff
@@ -44,24 +45,113 @@
-                <p>link10 <img src="cat.jpg" class="streamers"></p><img src="cat.jpg" class="streamer">
-                <p>link11 <img src="cat.jpg" class="streamers"></p><img src="cat.jpg" class="streamer">
-                <p>link12 <img src="cat.jpg" class="streamers"></p><img src="cat.jpg" class="streamer">
-                <p>link13 <img src="cat.jpg" class="streamers"></p><img src="cat.jpg" class="streamer">
-                <p>link14 <img src="cat.jpg" class="streamers"></p><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers" id="str"><p>link1</p><p  >5.6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers"><p>link2</p><p>0</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers" id="str"><p>link3</p><p>50k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers" id="str"><p>link4</p><p>6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers"><p>link5</p><p>1.4k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers"><p>link6</p><p>5.6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers"><p>link7</p><p>5.6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers" id="str"><p>link8</p><p>5.6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers" id="str"><p>link9</p><p>5.6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers" id="str"><p>link10</p><p>5.6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers"><p>link11</p><p>5.6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers"><p>link12</p><p>5.6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers"><p>link13</p><p>5.6k</p></div><img src="cat.jpg" class="streamer">
+                <div class="foll"><img src="cat.jpg" class="streamers"><p>link14</p><p>5.6k</p></div><img src="cat.jpg" class="streamer">
             </div>
         </div>
+        <div id="left_down">
+            <div>
+                <i class="fa-brands fa-instagram" id="icons"></i>
+                <i class="fa-brands fa-telegram" id="icons"></i>
+                <i class="fa-brands fa-twitter" id="icons"></i>
+            </div>
+        </div>
+        <main>
+            <div class="video">
+                <i class="fa-solid fa-paw fa-xl" id="paw_video"></i>
```

Comment:
Id must be unique, use class

Reply:
I tried, and it didn't work when I put "class".

Reply:
<i class="fa-solid fa-paw fa-xl paw_video"></i> ?
and try to use !important in styles

Reply:
it is working if use !important